db/seed/beneficiary.seed.ts

```diff
@@ -0,0 +1,45 @@
+import { PrismaClient, PersonRelation } from '@prisma/client'
+const prisma = new PrismaClient()
+
+export async function beneficiarySeed() {
+  console.log('Beneficiary seed')
+
+  const persons = await prisma.person.findMany({ where: { newsletter: true } })
+  console.log(persons)
+
+  if (!persons) {
+    throw new Error('No persons subscribed to newsletter')
+  }
+
+  const bg = await prisma.country.findFirst({ where: { countryCode: 'BG' } })
+  console.log(bg)
+
+  if (!bg) {
+    throw new Error('No country BG')
+  }
+
+  const cityFromDb = await prisma.city.findFirst({ where: { countryId: bg.id} })
+  console.log(cityFromDb)
+
+  if (!cityFromDb) {
+    throw new Error('No city')
+  }
+
+  const coordinatorFromDb = await prisma.coordinator.findFirst()
+  console.log(coordinatorFromDb)
+
+  if (!coordinatorFromDb) {
+    throw new Error('No coordinator')
+  }
+
+  var result= new Array();
```

Comment:
Usage of `const` is advised over `var` or `let` to provide immutability. There's no problem adding items to the array as the type of the variable is not being changed.

```suggestion
  const result= []
```